tools/shoestring/tests/resources/sai.shoestring.ini

```diff
@@ -27,6 +27,7 @@ lockedFundsPerAggregate = 10000000
 
 harvester =
 voter =
+node_key =
```

Comment:
unsure if naming would be more consistent as `node`

Reply:
Yea, I was thinking about this but was not sure.

``node`` is so generic and does not say what about the node is getting imported.
maybe just ``key``?  or ``cert`` and point to the cert folder?